app/Mail/GeneralInterestInvite.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace App\Mail;
+
+use Illuminate\Bus\Queueable;
+use Illuminate\Mail\Mailable;
+use Illuminate\Queue\SerializesModels;
+use Illuminate\Contracts\Queue\ShouldQueue;
+
+class GeneralInterestInvite extends Mailable
+{
+    use Queueable, SerializesModels;
+
+    /**
+     * Create a new message instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        //
+    }
+
+    /**
+     * Build the message.
+     *
+     * @return $this
+     */
+    public function build()
+    {
+        $this->from('noreply@my.robojackets.org', 'RoboJackets')
+             ->subject('RoboJackets General Interest Meeting - RSVP Requested')
+             ->markdown('mail.generalinterest.invite');
+
+        $this->withSwiftMessage(function ($message) {
+            $message->getHeaders()
+                    ->addTextHeader('Reply-To', 'RoboJackets Support <support@robojackets.org>');
```

Comment:
We could leave off the Reply-To. As a general rule, nobody should be replying to this email, but if they do I want it to go to the admin mailbox, hence the setup.

Alternatively we can send from support@my, which also goes to the admin mailbox.